### ISSUE TITLE
Continuous latent loss possibly wrong in InfoGAN

### ISSUE THREAD

**Issue by abdulfatir** [hwalsuklee/tensorflow-generative-model-collections#21]
In [InfoGAN.py line 146](https://github.com/hwalsuklee/tensorflow-generative-model-collections/blob/master/infoGAN.py#L148), the squared loss is being optimized as follows.
```python
cont_code_est = code_fake[:, self.len_discrete_code:]
cont_code_tg = self.y[:, self.len_discrete_code:]
q_cont_loss = tf.reduce_mean(tf.reduce_sum(tf.square(cont_code_tg - cont_code_est), axis=1))
```
The `code_fake` vector is in **[0,1]** as it comes from a softmax non-linearity. The actual latent code being sent, however, is in **[-1,1]**. (See [line 234](https://github.com/hwalsuklee/tensorflow-generative-model-collections/blob/master/infoGAN.py#L234)).
```python
batch_codes = np.concatenate((batch_labels, np.random.uniform(-1, 1, size=(self.batch_size, 2))),
                                             axis=1)
```

Am I missing something?

**Comment by hwalsuklee** [comment]
Oh..

You are right. It must be `code_logit_fake`.

Thanks for your contribution.

I've changed code.
